source/github-bugs.css

```
@@ -0,0 +1,30 @@
+/* Place Releases/Tags selector on the same line as `Draft new release` on releases page (https://github.com/sindresorhus/refined-github/commit/a34981cb3f0a1cf833f6e8f9f9591427d8f96473) */
```

Seems like that PR's author deleted their account so we only have commit link left 🤷‍♀️

---

This is the issue where I originally mentioned the bug (second comment)
```suggestion
/* Place Releases/Tags selector on the same line as `Draft new release` on releases page #2325 */
```